attestation-service/rvps/src/bin/rvps.rs

```diff
@@ -19,7 +19,7 @@ const DEFAULT_STORAGE: &str = "LocalFs";
 #[command(author, version, about, long_about = None)]
 pub struct Cli {
     /// Underlying storage engine that RVPS uses.
-    #[arg(short, long, default_value = DEFAULT_STORAGE)]
+    #[arg(short = 'c', long, default_value = DEFAULT_STORAGE)]
```

Comment:
Why use `-c` for short instead of `-s`?

Reply:
By default both `storage` and `socket` will use the same short flag `-s` atomatically, which will make the rvps process panic when launched. This is why a different short flag is defined here.